framework/messages/de/yii.php

```diff
@@ -56,32 +56,32 @@
     'The requested view "{name}" was not found.' => 'Die View-Datei "{name}" konnte nicht gefunden werden.',
     'The verification code is incorrect.' => 'Der Prüfcode ist falsch.',
     'Total <b>{count, number}</b> {count, plural, one{item} other{items}}.' => 'Insgesamt <b>{count, number}</b> {count, plural, one{Eintrag} other{Einträge}}.',
-    'Unable to verify your data submission.' => 'Es ist nicht möglich, Ihre Dateneingabe zu prüfen.',
+    'Unable to verify your data submission.' => 'Dateneingabe konnte nicht überprüft werden oder ist ungültig.',
     'Unknown alias: -{name}' => 'Unbekannter Alias: -{name}',
     'Unknown option: --{name}' => 'Unbekannte Option: --{name}',
     'Update' => 'Bearbeiten',
     'View' => 'Anzeigen',
     'Yes' => 'Ja',
     'Yii Framework' => 'Yii Framework',
-    'You are not allowed to perform this action.' => 'Sie dürfen diese Aktion nicht durchführen.',
     'You can upload at most {limit, number} {limit, plural, one{file} other{files}}.' => 'Sie können maximal {limit, plural, one{eine Datei} other{# Dateien}} hochladen.',
+    'You are not allowed to perform this action.' => 'Diese Aktion ist ungültig.',
```

Comment:
Existing one is correct. This one would actually translate to `This action is invalid."